microshift_support/microshift-version.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+[id="microshift-version"]
+= MicroShift Version
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-version
+toc::[]
+
+[id="microshift-version-cli"]
+== Checking the MicroShift Version from the command line
+
+From the command line, use `microshift version` to check the version information.
+
+[source,terminal]
+----
+$ microshift version
+----
+
+.Example output
+[source,terminal]
+----
+MicroShift Version: 4.10.0-0.microshift-e6980e25
+Base OCP Version: 4.10.18
+----
+
+[id="microshift-version-api"]
+== Checking the MicroShift Version via the API
+
+Through the API, access the `kube-public/microshift-version` ConfigMap to retrieve the same information.
+
+[source,terminal]
+----
+$ oc get configmap -n kube-public microshift-version -o yaml
+----
+
+.Example output
+[source,terminal]
```

Comment:
Would it make more sense to use the YAML syntax highlighting here?

Reply:
Yes, good point.